app/views/eventDetails/paymentOptions.html

```diff
@@ -5,6 +5,15 @@ <h2 class="page-title border" translate>Payment Options</h2>
     <h3 class="panel-title" translate>Credit Card</h3>
   </div>
   <div class="panel-body">
+    <div class="form-group">
+      <div class="col-sm-12 stacked-spacing-col-sm" ng-if="conference.paymentGatewayType === 'AUTHORIZE_NET'">
+        <label translate>Select A Payment Gateway Type:</label>
+        <select class="form-control" ng-model="conference.paymentGatewayType">
```

Comment:
I think that is a good suggestion (less code).  I made the change, but waiting to hear back from Lee as the link I used to use at the bottom of the https://stage.eventregistrationtool.com/#/ site is now missing (i.e. ‘Local Dev Environment’).
I would always click on this link after login to Cru Relay to do dev/test locally.
Waiting to push changes until this is resolved.